CHANGELOG.md

```diff
@@ -8,6 +8,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ## [Unreleased]
 
 ### Added
+- Updated corrent port in demo meeting documentation. Made demo meeting dev server easy to run. 
```

Comment:
You can drop this since this is a demo change. The `CHANGELOG` addition is necessary for the changes made to library.

Reply:
when I don't make this change, `npm run build:release` command fails with error stating no change found in CHANGELOG, is that fine?

Reply:
Hmm, @michhyun1 @nicklera1 can you check this once?

Reply:
The build has passed.

Reply:
Yeah the build passed.